[FIX] sign-up 페이지 TERMS_CONFIG import 오류 수정

## 📑 Description

PR #243에서 `terms.config.ts` 삭제 후 `sign-up/page.tsx`에서 `TERMS_CONFIG` import 오류 발생

**원인:**
- PR #230에서 `shared/constants/terms/terms.config.ts` 추가 및 `TERMS_CONFIG` export
- PR #243에서 해당 파일 삭제 → `TERMS_CONFIG` 사라짐
- `sign-up/page.tsx`에서 여전히 `@/shared`에서 `TERMS_CONFIG` import 시도

**빌드 에러:**
```
Module not found: Can't resolve '@/shared'
The export TERMS_CONFIG was not found in module
```

## 📌 To Do

- [ ] `sign-up/page.tsx`에서 약관 데이터 직접 import 방식으로 변경
- [ ] `shared/constants/terms/data/` 에서 직접 데이터 가져오도록 수정
- [ ] 빌드 테스트 확인

## 💬 Comment

관련 PR: #230, #243

--- a/omechu-app/src/app/(auth)/sign-up/page.tsx
+++ b/omechu-app/src/app/(auth)/sign-up/page.tsx
@@ -15,13 +15,17 @@ import {
   type SignupFormValues,
   useAuthStore,
 } from "@/entities/user";
+import { BottomButton, Header, ModalWrapper, Toast } from "@/shared";
 import {
-  BottomButton,
-  Header,
-  ModalWrapper,
-  Toast,
-  TERMS_CONFIG,
-} from "@/shared";
+  termsForServiceMain,
+  termsForServiceServe,
+  termsForPersonalInfoMain,
+  termsForPersonalInfoServe,
+  termsForLocationServiceMain,
+  termsForLocationServiceServe,
+  type TermsConfig,
+  type TermsType,
+} from "@/shared/constants/terms";
 import {
   SignUpForm,
   TermsModal,
@@ -29,6 +33,21 @@ import {
   MODAL_TO_TERMS_TYPE,
   MODAL_TO_FORM_FIELD,
 } from "@/widgets/auth";
+
+const TERMS_CONFIG: Record<TermsType, TermsConfig> = {
+  service: {
+    title: "서비스 이용약관",
+    data: [...termsForServiceMain, ...termsForServiceServe],
+  },
+  "personal-info": {
+    title: "개인정보 처리방침",
+    data: [...termsForPersonalInfoMain, ...termsForPersonalInfoServe],
+  },
+  "location-info": {
+    title: "위치기반 서비스 이용약관",
+    data: [...termsForLocationServiceMain, ...termsForLocationServiceServe],
+  },
+};
 
 export default function SignUpPage() {
   const [activeModal, setActiveModal] = useState<ModalType | null>(null);

